fluent-package/msi/assets/fluent-gem.bat

```diff
@@ -1,12 +1,12 @@
 @echo off
 if "%~nx0" == "td-agent-gem.bat" (
-  set FLUENT_PACKAGE_TOPDIR=%~dp0..\
+  set "FLUENT_PACKAGE_TOPDIR=%~dp0..\"
 ) else (
-  set FLUENT_PACKAGE_TOPDIR=%~dp0
+  set "FLUENT_PACKAGE_TOPDIR=%~dp0"
 )
-set FLUENT_PACKAGE_BINDIR=%FLUENT_PACKAGE_TOPDIR%bin
-set PATH="%FLUENT_PACKAGE_BINDIR%";%PATH%
-for /f "usebackq" %%i in (`%FLUENT_PACKAGE_BINDIR%\ruby.exe -rrbconfig -e "print RbConfig::CONFIG['ruby_version']"`) do set RUBY_VERSION=%%i
-set GEM_HOME=%FLUENT_PACKAGE_TOPDIR%lib\ruby\gems\%RUBY_VERSION%
-set GEM_PATH=%FLUENT_PACKAGE_TOPDIR%lib\ruby\gems\%RUBY_VERSION%
-%FLUENT_PACKAGE_BINDIR%\fluent-gem %*
+set "FLUENT_PACKAGE_BINDIR=%FLUENT_PACKAGE_TOPDIR%bin"
+set PATH=%FLUENT_PACKAGE_BINDIR%;%PATH%
+for /f "delims=" %%i in ('^""%FLUENT_PACKAGE_BINDIR%\ruby.exe" -rrbconfig -e "print RbConfig::CONFIG['ruby_version']"^"') do set RUBY_VERSION=%%i

```

Comment:
Sorry for my late response.
It looks good to me except for this line.

`for` of `batch` is very difficult for me. I read https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/for.

My understanding is that using `usebackq` is not wrong, and we don't need to specify `delims` for the purpose of this PR.

What we need would be to surround the entire command with `^"`.

```suggestion
for /f "usebackq" %%i in (`^""%FLUENT_PACKAGE_BINDIR%\ruby.exe" -rrbconfig -e "print RbConfig::CONFIG['ruby_version']"^"`) do set RUBY_VERSION=%%i
```

If we don't use `usebackq`, then this fix would be enough for this PR.

```suggestion
for /f %%i in ('^""%FLUENT_PACKAGE_BINDIR%\ruby.exe" -rrbconfig -e "print RbConfig::CONFIG['ruby_version']"^"') do set RUBY_VERSION=%%i
```

Let me know if I missed something.